installer/bundle/bundle_skel.sh

```diff
@@ -369,8 +371,7 @@ compare_install_type()
 
 python_ctypes_installed() {
     # Check for Python ctypes library (required for omsconfig)
-    hasCtypes=1
-    echo "Checking for ctypes python module ..."
```

Comment:
Why did you remove this message? It's not a  critical thing to print; I'm just curious.